app/controllers/geo_controller.rb

```diff
@@ -0,0 +1,26 @@
+class GeoController < ApplicationController
+  include Blacklight::Base
+  copy_blacklight_config_from(CatalogController)
```

Comment:
Yes, but it also carries over all the catalog controller actions - index/show/etc.